### ISSUE TITLE
Applications not starting

### ISSUE THREAD

**Issue by rugbroed** [Simmesimme/Gnome-Pie#171]
I've tried both versions, ie. the one with the standard repositories and the one from your testing respository. The standard repository crashes when adding applications and the one from your testing repository does not launch the applications added. 

I'm on Ubuntu 18.04.03.

If there is a log somewhere, I can send it to you.

Also, is there any plans to make gnome-pie into a proper gnome extension?

**Comment by Simmesimme** [comment]
Both versions should be pretty similar. Version 0.7.2, I guess. Are you on X11 or on Wayland? In anyway, you could try to run gnome-pie from the command line. Just exit it and type "gnome-pie" into a terminal. Is there any error printed?

**Comment by rugbroed** [comment]
Thanks for the tip :-)

I'm using X11.

I saw in the output that the programs were registered as files, not as executables. I had dragged them into Gnome-pie, and assumed that it would know they were programs. So I changed them to "Launch application" instead and it works now. Sorry about that!

It might be a good idea to alert the user to problems/warnings in the GUI - for example this warning; "[WARNING] No application is registered as handling this file"

Thanks!
